pubspec.yaml

```
@@ -13,7 +13,7 @@ publish_to: 'none'
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 1.8.0
+version: 1.8.1
```

We should be incrementing a minor version number here. We reserve patch version numbers for hot fixes.
```suggestion
version: 1.9.0
```

---

My bad